ledger-tool/src/main.rs

```diff
@@ -1203,13 +1209,28 @@ fn main() {
                     .value_name("SLOT")
                     .help("Ending slot to stop purging (inclusive) [default: the highest slot in the ledger]"),
             )
+            .arg(
+                Arg::with_name("batch_size")
+                    .long("batch-size")
+                    .value_name("NUM")
+                    .takes_value(true)
+                    .default_value("1000")
```

Comment:
well, I preferred more large values; but it seems that 10000 (45 slots/sec) worsens the throughput...

not much digging; I'll just settle down on 1000 (measured 50 slots/sec).